data/_ui/interfaces.txt

```diff
@@ -1298,6 +1299,88 @@ interface "bank"
 
 
 
+interface "shop"
+	anchor bottom right
+	value "fleet bar width" 250
+	value "item bar width" 300
+	box "find button"
+		center -580 -20
+		dimensions 26 26
+	box "leave button"
+		center -45 -25
+		dimensions 70 30
+	box "park button"
+		center -125 -25
+		dimensions 70 30
+	box "buy button"
+		center -505 -25
+		dimensions 70 30
+	box "sell button"
+		center -425 -25
+		dimensions 70 30
+	box "sell ship button"
+		center -215 -25
+		dimensions 50 30
```

Comment:
```suggestion
		dimensions 70 30
```

Reply:
you'd also need to move the centre of the box by 10px to compensate for the 20px wider button

Reply:
I have to go to work now. I'll try and position the wider buttons tonight.

Reply:
It looks like they are already slightly offset compared to the buttons on the right side, so maybe they were positioned for their full size?